vespa/package.py

```diff
@@ -418,7 +419,7 @@ def _run_vespa_engine_container(
                     hostname=application_name,
                     privileged=True,
                     volumes={disk_folder: {"bind": "/app", "mode": "rw"}},
-                    ports={self.local_port: self.local_port, 19112: 19112},
+                    ports={8080: self.local_port},
```

Comment:
So this is saying map container port 8080 to `self.local_port` right? If so, that looks ok.

Reply:
Yes. This was non-intuitive for me. I would assume the opposite direction but got here through trial and error.